_docs_integrate/data-model-overview.md

```diff
@@ -428,7 +430,9 @@ RequestItems can be sent inside of a [Request](#request) and specify what should
 | mustBeAccepted        | `boolean`                     | The mandatory `mustBeAccepted` property is used to differentiate between required and optional RequestItems within the Request.<br>{::nomarkdown}<ul><li>If <code>mustBeAccepted</code> is set to <code>true</code>, the peer cannot <a href="https://enmeshed.eu/use-case-consumption-accept-incoming-request">accept the Request</a> without accepting this RequestItem.</li><li>If <code>mustBeAccepted</code> is set to <code>false</code>, the peer can <a href="https://enmeshed.eu/use-case-consumption-accept-incoming-request">accept the Request</a> even though the RequestItem is rejected.</ul>{:/} |
 | requireManualDecision | `boolean` \| `undefined`      | To block the automated acceptance of Requests, the requireManualDecision property can be set to true. The default is, that each RequestItem may be automatically processed on the peer side. If the sender would like to have an enforced manual acceptance step the requireManualDecision property can be set to true.                                                                                                                                                                                                                                                                                          |
 
-#### [ConsentRequestItem]({% link _docs_integrate/request-and-response-introduction.md %}#consentrequestitem)
+#### ConsentRequestItem
+
+For more information you should checkout the section [ConsentRequestItem]({% link _docs_integrate/request-and-response-introduction.md %}#consentrequestitem) of the [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}).
```

Comment:
Please also link to the page `request-one-time-consent-of-peer` here.

Reply:
I didn't add it because it is also linked in the section where I linked to, but if you think it would be better to add the link there directly, I will do so.